grails-app/views/stockMovement/_list.gsp

```diff
@@ -91,7 +91,7 @@
                     <g:formatDate format="MMM dd, yyyy" date="${stockMovement?.dateRequested}"/>
                 </td>
                 <td>
-                    <g:formatDate format="MMM dd, yyyy" date="${stockMovement?.dateCreated}"/>
+                    <g:formatDate format="MMM dd, yyyy" date="${stockMovement?.dateCreated ?: stockMovement?.requisition?.dateCreated}"/>
```

Comment:
@DominikaHatala This would be what I would consider an implementation leak. We actually want to update the dateCreated on the stock movement object. 

I'll try to deal with this myself but if I don't finish by tomorrow can you look into it. In particular I would look into the builder methods (createFromRequisition and createFromShipment) to see why the dateCreated is not populated correctly.